README.md

```diff
@@ -17,9 +20,22 @@ This chart will deploy the following on the target OpenShift cluster:
 - [PostgreSQL](https://www.postgresql.org/) database is available with credentials to manage the tablespace (optional).
   - A [reference implementation](#postgresql-deployment-reference-implementation) is provided for your convenience.
 - A Github API Token - in order to import items into the catalog, there is a need for GITHUB_TOKEN with the permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions: repo (all), admin:org (read:org) and user (read:user, user:email).
-  
+- `ArgoCD/Red Hat GitOps` operator is installed and one instance of `ArgoCD` exists in a given namespace (later referenced by `ARGOCD_NAMESPACE` env var)
+  - Validated API is `argoproj.io/v1alpha1/AppProject`
+- `Tekton/Red Hat Pipelines` operator is installed in the orchestrator namespace (e.g. `orchestrator.namespace` release value)
```

Comment:
And how those operators are installed?
Could those operators be installed by the charts as well, like it is done for the others operators?

Reply:
We decided for now to avoid this step and simplify the chart development

Reply:
ok

Reply:
s/Red Hat Pipelines/OpenShift Pipelines

Reply:
Why OpenShift Pipeline should be installed in a specific namespace?
If it is all-namespaces operator, wouldn't it be fine?